src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,61 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("String 클래스에 대한 학습 테스트")
+public class StringTest {
+
+  @DisplayName("쉼표를 구분자로 split했을 때 올바르게 분리된다")
+  @Test
+  void splitByComma() {
+    String[] result = "1,2".split(",");
+
+    assertThat(result).containsExactly("1", "2");
+  }
+
+  @DisplayName("구분자가 없는 문자열을 split했을 때 하나의 요소만 반환한다")
+  @Test
+  void splitWithoutComma() {
+    String[] result = "1".split(",");
+
+    assertThat(result).containsExactly("1");
+  }
+
+  @DisplayName("substring으로 괄호를 제거하면 내부 문자열만 반환한다")
+  @Test
+  void removesParentheses() {
+    String input = "(1,2)";
+
+    String result = input.substring(1, input.length() - 1);
+
+    assertThat(result).isEqualTo("1,2");
+  }
+
+  @Test
+  @DisplayName("charAt으로 특정 인덱스의 문자를 반환한다")
+  void returnsCharacterAtSpecificIndex() {
+    String input = "abc";
+    int targetIndex = 2;
+
+    char result = input.charAt(targetIndex);
+
+    assertThat(result).isEqualTo('c');
+  }
+
+  @Test
+  @DisplayName("charAt에 범위를 벗어난 인덱스를 전달하면 StringIndexOutOfBoundsException이 발생한다")
+  void throwsExceptionWhenIndexOutOfBounds() {
+    String input = "abc";
+    int outOfBoundIndex = 10;
+
+    assertThatThrownBy(
+            () -> {
+              input.charAt(outOfBoundIndex);
```

Comment:
```suggestion
    assertThatThrownBy(
            () -> {
              "abc".charAt(3);
```
로컬 변수가 재사용되지도 않는데 로컬 변수 없이 구현하는 것은 어떨까?
10보다는 경계값에 해당하는 3은 어떨까?

Reply:
피드백 감사합니다!
경계값을 명시적으로 설정하는 방식으로 개선하겠습니다!